.github/workflows/integrated-test.yml

```diff
@@ -0,0 +1,78 @@
+name: Retest Integrated Tests
+
+on:
+  workflow_call:
+    inputs:
+      checkout-repo:
+        required: false
+        type: string
+      checkout-ref:
+        required: false
+        type: string
+      build-arch:
+        required: true
+        type: string
+      pr-number:
+        required: false
+        type: string
+      sha-value:
+        required: false
+        type: string
+      skip-itests:
+        required: false
+        type: boolean
+        default: false
+
+permissions:
+  contents: read
+  packages: read
+
+jobs:
+  integration-tests:
+    runs-on: ubuntu-latest
+    if: ${{ !inputs.skip-itests }}
+    steps:
+    - name: Install xpath
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y libxml-xpath-perl
+    - name: Install qemu
+      if: ${{ inputs.build-arch != 'amd64' }}
```

Comment:
Maybe this can be refactored? It ends up looking like this in pseudocode terms:

```
ci(false)
ci(skip) {
  run_tests(skip)
}
run_tests(skip) {
  if (skip) return
  ... # do tests
}
```

But perhaps it could just be:

```
ci(false)
ci(skip) {
  if (!skip) run_tests()
}
run_tests() {
  ... # do tests
}
```

ie. whoever is calling this workflow can just decide not to call it, rather than passing in this parameter that makes the whole thing do nothing?

Reply:
The updated code uses the default values of the inputs